aten/src/ATen/DLConvertor.cpp

```diff
@@ -298,6 +298,16 @@ Tensor fromDLPack(DLManagedTensor* src) {
   return fromDLPack(src, std::move(deleter));
 }
 
+Tensor fromDLPack(const DLManagedTensor* src) {
+  auto deleter = [src](void* self) {
+    if (src->deleter) {
+      // NOLINTNEXTLINE(cppcoreguidelines-pro-type-const-cast)
+      src->deleter(const_cast<DLManagedTensor*>(src));
+    }
+  };
+  return fromDLPack(src, std::move(deleter));
```

Comment:
Do not copy the code, just cost cast a top level wrapper
```suggestion
      // NOLINTNEXTLINE(cppcoreguidelines-pro-type-const-cast)
     return from DLPack(const_cast<DLManagedTensor*>(src));
```